README.md

```diff
@@ -351,7 +351,7 @@ webhooks.on(eventNames, handler)
     </td>
     <td>
       <strong>Required.</strong>
-      Name of the event. One of <a href="#listofalleventnames">GitHub’s supported event names</a>.
+      Name of the event. One of [GitHub’s supported event names](https://developer.github.com/webhooks/#events).
```

Comment:
Markdown won’t render correctly

![image](https://user-images.githubusercontent.com/39992/59579788-5dc7c700-9083-11e9-87b6-c54338ccf8e5.png)

I think it’s because of the indentation. If you remove the indentation and add empty lines above and below, then it works. But I think it remains more readable when you stick with the `<a>` tags

Reply:
I've reverted it back to anchor tags, should be good now.

I still question why we can't use `target="_blank"` and `rel="noopener noreferrer"` - These are very common and a recommended [security practice](https://developers.google.com/web/tools/lighthouse/audits/noopener)

Reply:
The docs say

> In general, when you use `target="_blank"`, always add `rel="noopener"` or `rel="noreferrer"`.

So if we don’t add `target="_blank"` we should be good?